op-conductor/health/monitor.go

```diff
@@ -182,10 +222,34 @@ func (hm *SequencerHealthMonitor) healthCheck(ctx context.Context) error {
 		return ErrSequencerNotHealthy
 	}
 
-	hm.log.Info("sequencer is healthy")
 	return nil
 }
 
+func (hm *SequencerHealthMonitor) checkRollupBoost(ctx context.Context) error {
+	// Skip the check if rollup boost client is not configured
+	if hm.rb == nil {
+		hm.log.Info("rollup boost client is not configured, skipping health check")
+		return nil
+	}
+
+	status, err := hm.rb.Healthcheck(ctx)
+	if err != nil {
+		hm.log.Error("health monitor failed to get rollup boost status", "err", err)
+		return ErrRollupBoostConnectionDown
+	}
+
+	switch status {
+	case client.HealthStatusHealthy:
+		return nil
+	case client.HealthStatusPartial:
```

Comment:
Suggestion to add logs in the event there is a non-healthy status returned

Reply:
Added in ad26b6357c4683839cf83e85731e160b598c13f5.